### ISSUE TITLE
Highlight for current selected item/line in symbol reference location window

### ISSUE THREAD

**Issue by ray-x** [neoclide/coc.nvim#2080]
**Is your feature request related to a problem? Please describe.**
In symbol reference location window, the current selected symbol reference do not have dedicated highlight. It will reuse the highlight of CursorLine. But in lot of the dark color scheme, the CursorLine is close to the background (less distraction, also,  it is more important to see the CursorColumn instead of CursorLine)
This makes it very hard to see which symbol reference was selected.
Following examples are from OneDark and Plastic.  Both are not easy to tell at first glance which symbol was selected. 

<img width="581" alt="Screen Shot 2020-06-16 at 3 11 00 pm" src="https://user-images.githubusercontent.com/1681295/84734933-95928300-afe5-11ea-839e-390cdea403e8.png">
<img width="359" alt="Screen Shot 2020-06-16 at 3 32 37 pm" src="https://user-images.githubusercontent.com/1681295/84735358-abed0e80-afe6-11ea-9943-4d3ee4dc4e2a.png">

**Describe the solution you'd like**
I would suggested either use PMenuSel as the selected item. Or provided a way user can configure the selected items, like CocLocationSel.

**Describe alternatives you've considered**
Provide a sign to indicate the highlighted item. Change to Bold etc....

**Comment by chemzqm** [comment]
Change your CursorLine highlight in your vimrc of use preview window.

**Comment by ray-x** [comment]
I do know that change CursorLine is one  walk-around. But still I do not like the CursorLine to be too much different compared to background. And that is why I will always customer the Cursor highlight for theme. One dark is one exception.
Also logically, selected line and cursor line are different. And it really took me a while to figure out the highlight is using CursorLine. 
Preview is another story. Normally, I will focus on the symbol list window first and select the symbol location, only after that I will use preview window.

**Comment by dseeni** [comment]
Is there any way to change the highlight line for coc preview location (shows the current reference in context.. something changed in the past 6 months and I can't see the white text with yellow background'.. can I change the blue highlight to something else for the current highlighted reference???

![image](https://user-images.githubusercontent.com/6591429/106518517-d9f86380-648e-11eb-9a9a-e54a438bd586.png)

**Comment by chemzqm** [comment]
Already using `Search` highlight group for selected range on latest locations list of coc.nvim.
